ios/app/brave_core_main.mm

```diff
@@ -255,6 +260,19 @@ - (BraveSyncAPI*)syncAPI {
   return _syncAPI;
 }
 
+- (BravePasswordAPI*)passwordAPI {
+  if (!_passwordAPI) {
+    scoped_refptr<password_manager::PasswordStoreInterface> password_store_ =
```

Comment:
another security consideration is that the synced passwords must never be stored in plaintext on disk anywhere and if possible, they should be purged from memory when no longer needed

Reply:
The passwords on iOS will not be stored anywhere disk or otherwise. They will be fetched, updated or deleted using the interface we created from Chromium whenever they are needed.